src/main/java/frc/team2412/robot/Robot.java

```diff
@@ -124,7 +125,7 @@ public void startCompetition() {
 
     @Override
     public void robotInit() {
-        LiveWindow.setEnabled(false);
+        // LiveWindow.setEnabled(false);
```

Comment:
Why was this commented out? Is this supposed to be permament? If so, please delete this and now-unused `LiveWindow` import.

Reply:
It removed PDP logging, undid it cuz we wanted that